.github/workflows/tests.yml

```diff
@@ -597,6 +597,7 @@ jobs:
           python=${{ matrix.python-version }}
 
       - name: Conda Remove anaconda-auth
+        if: matrix.arch != 'osx-64'
```

Comment:
thanks, we may also remove this conda remove again (after the tests have time to run, pass, and get merged) if the culprit is not these packages but the failing get_index_lazy test.

Reply:
It looks like we may have another problem with mac osx-64.  It seems it can't find it.  I also just noticed that conda only does arm atm.  Should we just drop osx-64 testing altogether?